### PR TITLE
[PXC-869] Show messages if mysql_upgrade is needed

### PR DESCRIPTION
**Pull request by EvgeniyPatlan**

### CODE REVIEW

**Review by hrvojem — APPROVED** [review]
LGTM

File: build-ps/debian/percona-xtradb-cluster-server-5.7.preinst [hunk 1]
```diff
@@ -152,6 +152,24 @@ case "$1" in
 
 	#DEBHELPER#
 	server_stop
+	currentver=$(dpkg -l | grep percona-xtradb-cluster-server-5.7 | awk '{print $3}' | awk -F'-' '{print $1}')
+	requiredver="5.7.19"
+	if [[ $currentver < $requiredver ]]; then
+cat >/usr/share/mysql/pxc_cluster_view.sql << EOL
```
Comment (dbadapt) [inline build-ps/debian/percona-xtradb-cluster-server-5.7.preinst:158]: Why is the file in a heredoc?   Can't we just commit the file into the repo and install it with the package?

File: build-ps/percona-xtradb-cluster.spec [hunk 2]
```diff
@@ -1311,6 +1321,22 @@ fi
 if [ -x sbin/restorecon ] ; then
   sbin/restorecon -R var/lib/mysql
 fi
+if [ "$UPDATE_REQUIRES" = "true" ] ; then
+cat >/usr/share/mysql/pxc_cluster_view.sql << EOL
```
Comment (dbadapt) [inline build-ps/percona-xtradb-cluster.spec:1325]: likewise,  why a heredoc?  Just commit pxc_cluster_view.sql and install as a package managed file.

File: build-ps/ubuntu/percona-xtradb-cluster-server-5.7.preinst [hunk 3]
```diff
@@ -153,6 +152,24 @@ case "$1" in
 
 	#DEBHELPER#
 	server_stop
+	currentver=$(dpkg -l | grep percona-xtradb-cluster-server-5.7 | awk '{print $3}' | awk -F'-' '{print $1}')
+	requiredver="5.7.19"
+	if [[ $currentver < $requiredver ]]; then
+cat >/usr/share/mysql/pxc_cluster_view.sql << EOL
```
Comment (dbadapt) [inline build-ps/ubuntu/percona-xtradb-cluster-server-5.7.preinst:158]: why heredoc?

**Pull request merged**